src/System.Management.Automation/commands/utility/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -713,6 +718,11 @@ public abstract class PSControl
         /// </summary>
         public bool OutOfBand { get; set; }
 
+        /// <summary>
+        /// Set explicit limit for Width. Default width is a console host width.
+        /// </summary>
+        public int? Width { get; set; }
+
```

Comment:
New properties in the public Format api surface require special code in deserialization for remoting - see [here](https://github.com/PowerShell/PowerShell/blob/8731747c3e412edf537c3dc0b4ce6a09e6fd7d95/src/System.Management.Automation/engine/serialization.cs#L7382)

Note that you need to use `RehydrationFlags.MissingPropertyOk`, otherwise remoting interop with Windows PowerShell will not work.

It's possible you might also need to update [this method](https://github.com/PowerShell/PowerShell/blob/8731747c3e412edf537c3dc0b4ce6a09e6fd7d95/src/System.Management.Automation/commands/utility/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs#L723) but it's possible serialization of this property won't have any effect on Windows PowerShell, so it might be harmless.